corehq/apps/repeaters/migrations/f_0001_adjust_auth_field_format.py

```diff
@@ -0,0 +1,37 @@
+from django.db import migrations
```

Comment:
Ah, I put it in there when I was doing some shell testing so that I could import this file (leading numbers in file names break the normal import workflow), but didn't intend to leave it. Updated.